Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -38,12 +38,12 @@ class PFPlaylist: PFObject, PFSubclassing {
     class func addSongtoPlaylistInBackground(song: PFSong, playlist: PFPlaylist, completion: @escaping ((Bool, Error?)-> Void)){
         // create an entry in the Follow table
         let joinTable = PFObject(className: "SongJoinTable")
-        joinTable.setObject(song, forKey: "Song")
-        joinTable.setObject(playlist, forKey: "Playlist")
+        joinTable.setObject(song, forKey: "song")
+        joinTable.setObject(playlist, forKey: "playlist")
         
         let currentDate = NSDate()
-        joinTable.setObject(currentDate, forKey: "AddedAt")
-        joinTable.setObject(currentDate, forKey: "LastPlayed")
+        joinTable.setObject(currentDate, forKey: "addedAt")
+        joinTable.setObject(currentDate, forKey: "lastPlayed")
```

Comment:
Is this true that whenever you add a song to playlist it is being played? Is there any view for a user to browse a list of songs and add them to a playlist?

Reply:
Currently the only functionality to add a song to a playlist is when it is playing. But, this is a good point if I want to implement it in the future